app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,151 @@
+- Storybook
+
+## Decision Outcome
+
+Chosen option: "[TBD]", because [TBD].
+
+LZoog 2/16: I've gone back and forth on what I think would be best and at the moment, I'm leaning towards the Docusaurus + GH Pages option, but would like input from those that will actually be using these docs before making the call. While the "cons" can be mitigated by clear getting started docs, I'm not sure how heavily weighted some pros and cons are, like writing in MD, using GH, editing and committing doc files within the repository, or peer reviews of doc edits.
+
+### Positive Consequences
+
+- TBD, a couple sentences from the "Pros" section of the chosen option
+
+Note: we could technically point any of these options to a custom domain if we wanted, even if we just wanted to use a vanity URL to redirect to the documentation (for the Wiki option).
+
+### Negative Consequences
+
+- TBD, a couple sentences from the "Cons" section of the chosen option
+
+## Pros and Cons of the Options
+
+### Mana
+
+With this option, we would create a "Documentation" child page [on Mana](https://mana.mozilla.org/wiki/display/FJT/Project+Nimbus) with a home page with a quick explanation of the Nimbus platform, linking out to relevant high-level docs, and creating individual pages for "User docs" (product managers, experiment owners), "Engineering docs," and "Data docs." May warrant some shuffling of other pages.
+
+Pros
+
+- A lot of documentation already exists on Mana and may be the most natural place for experiment owners to look for central docs; would be easily accessible
+- An email is sent out to those subscribed when anyone updates a document
+- Jira tickets can be easily linked by reference number
+- Some users may feel the most comfortable editing and maintaining in Mana rather than editing in GH or submitting a PR
+- No setup or deployment required
+
+Neutral
+
+- Documentation edits wouldn't require peer review and approval
+- Has a search option, but within Firefox Journey and not within Nimbus specifically
+
+Cons
+
+- May not be a natural place to look for engineers
+- [Supports Markdown](https://confluence.atlassian.com/doc/confluence-wiki-markup-251003035.html#ConfluenceWikiMarkup-markdownCanIinsertmarkdown?), but differs slightly from conventional Markdown and won't load separate files
+- Can be hard to read as the text expands across the entire page
+- Is not very flexible or customizable without paid add-ons
+- Is arguably the least open-source friendly
+
+### Docusaurus + GH Pages
+
+The [FxA Ecosystem Platform docs](https://mozilla.github.io/ecosystem-platform/) use [Docusaurus](https://docusaurus.io/). For this option, we would likely set up a `docs` or similar branch specifically for documentation in the Experimenter repo and we would point GH to push this branch to [GitHub pages](https://docs.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).
```

Comment:
BTW @jodyheavener going to add you as a decider since we briefly chatted about some of this in Slack👀